### PR TITLE
Update library doc page with 'Standard' field removed

### PR DESCRIPTION
**Pull request by novoio**
- Remove 'Standard' field from each library info
- Remove 'Standard Proposals' and 'TR1 libraries' from sort categories
- Add 'C++ Minimum' to sort categories

### CODE REVIEW

File: doc/libraries.php [hunk 1]
```diff
@@ -21,8 +21,8 @@ class LibraryPage {
     );
 
     static $filter_fields = Array(
-        'std-proposal' => 'Standard Proposals',
-        'std-tr1' => 'TR1 libraries',
+        // 'std-proposal' => 'Standard Proposals',
+        // 'std-tr1' => 'TR1 libraries',
```
Comment (glenfe) [inline doc/libraries.php:25]: Don't comment it out, just delete it

File: doc/libraries.php [hunk 2]
```diff
@@ -32,12 +32,14 @@ class LibraryPage {
         'boost-version' => 'First Release',
         'std-proposal' => 'STD Proposal',
         'std-tr1' => 'STD::TR1',
```
Comment (glenfe) [inline doc/libraries.php:34]: Shouldn't these two be removed?

**Review by glenfe — CHANGES_REQUESTED** [review]
If we're removing std-proposal and std-tr1, they should be removed in all places in this file and others.

File: doc/libraries.php (hunk 2 above)
Comment (novoio) [reply doc/libraries.php]: @glenfe Just done your feedback

**Comment by glenfe** [comment]
I pointed a few more places out in the channel:

* https://github.com/boostorg/website/blob/739c70cb172f41c03ac44e2e8497e3e6a5e29c61/doc/libraries.php#L245-L250
* https://github.com/boostorg/website/blob/739c70cb172f41c03ac44e2e8497e3e6a5e29c61/common/code/boost_utility.php#L59-L63
* https://github.com/boostorg/website/blob/739c70cb172f41c03ac44e2e8497e3e6a5e29c61/common/code/boost_libraries.php#L119-L129

**Pull request merged**